public/index.html

```diff
@@ -14,5 +14,6 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <script src="../src/client/tech.js"></script>
```

Comment:
I think this is left-overs from the session you had with Ed, it probably needs to be taken out before you merge this pull request.